core/app/[locale]/(default)/(auth)/login/page.tsx

```diff
@@ -19,37 +16,37 @@ export async function generateMetadata({ params }: Props) {
 }
 
 interface Props {
-  params: Promise<{ locale: string }>;
+  params: { locale: (typeof locales)[number] };
 }
 
-export default async function Login({ params }: Props) {
-  const { locale } = await params;
-
+export default function Login({ params: { locale } }: Props) {
   setRequestLocale(locale);
 
-  const t = await getTranslations('Login');
+  const t = useTranslations('Login');
 
   return (
-    <div className="mx-auto my-6 max-w-4xl">
-      <h2 className="text-h3 mb-8 text-3xl font-black lg:text-4xl">{t('heading')}</h2>
-      <div className="mb-12 grid grid-cols-1 lg:grid-cols-2 lg:gap-x-8">
-        <LoginForm />
-        <div className="flex flex-col gap-4 bg-gray-100 p-8">
-          <h3 className="mb-3 text-xl font-bold lg:text-2xl">{t('CreateAccount.heading')}</h3>
-          <p className="text-base font-semibold">{t('CreateAccount.accountBenefits')}</p>
-          <ul className="mb-4 list-disc ps-4">
-            <li>{t('CreateAccount.fastCheckout')}</li>
-            <li>{t('CreateAccount.multipleAddresses')}</li>
-            <li>{t('CreateAccount.ordersHistory')}</li>
-            <li>{t('CreateAccount.ordersTracking')}</li>
-            <li>{t('CreateAccount.wishlists')}</li>
-          </ul>
-          <Button asChild className="w-fit items-center px-8 py-2 hover:text-white">
-            <Link href="/register">{t('CreateAccount.createLink')}</Link>
-          </Button>
-        </div>
+    <SignInSection
+      action={login}
+      forgotPasswordHref="/login/forgot-password"
+      forgotPasswordLabel={t('Form.forgotPassword')}
+      submitLabel={t('Form.logIn')}
+      title={t('heading')}
+    >
+      <div className="">
+        <h3 className="mb-3 text-xl font-bold lg:text-2xl">{t('CreateAccount.heading')}</h3>
+        <p className="text-base font-semibold">{t('CreateAccount.accountBenefits')}</p>
+        <ul className="mb-4 list-disc ps-4">
+          <li>{t('CreateAccount.fastCheckout')}</li>
+          <li>{t('CreateAccount.multipleAddresses')}</li>
+          <li>{t('CreateAccount.ordersHistory')}</li>
+          <li>{t('CreateAccount.ordersTracking')}</li>
+          <li>{t('CreateAccount.wishlists')}</li>
+        </ul>
+        <ButtonLink href="/register" variant="secondary">
+          {t('CreateAccount.createLink')}
+        </ButtonLink>
       </div>
```

Comment:
This isn't essential for this PR but I think we should rename this prop from `children` to `marketing` or `promotion`

Reply:
Yeah, this component seems like a one off.

Reply:
Yeah we can revisit later.